gammapy/spectrum/flux_point.py

```diff
@@ -756,16 +756,18 @@ def compute_flux_point_sqrt_ts(self, fit, stat_best_fit):
     def fit_point(self, model, energy_group, energy_ref, sqrt_ts_threshold=1):
         from .fit import SpectrumFit
 
-        energy_min = energy_group.energy_min
-        energy_max = energy_group.energy_max
+ #       energy_min = energy_group.energy_min
+ #       energy_max = energy_group.energy_max
+
+        self.obs.on_vector.bins_in_safe_range = range(energy_group.bin_idx_min,energy_group.bin_idx_max)
 
         # Set reference and remove min amplitude
         model.parameters['reference'].value = energy_ref.to('TeV').value
 
         fit = SpectrumFit(self.obs, model)
 
         # TODO: Notice channels contained in energy_group
```

Comment:
Don't understand that TODO. (I know you didn't write it) Can it be tackled here or removed?

Reply:
It is actually tackled here